src/app/Caches/Create/LanguageToggleRadios.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { FormGroup, Radio } from '@patternfly/react-core';
+import { ConfigDownloadType } from '@services/infinispanRefData';
+import { useTranslation } from 'react-i18next';
+const LanguageToggleRadios = (props: {
+  language: ConfigDownloadType;
+  setLanguage: (ConfigDownloadType) => void;
+  setContentType: (string) => void;
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <FormGroup hasNoPaddingTop isInline label="Code language" fieldId="code-language-radio-field">
+      <Radio
+        name="language-radio"
+        id="JSON"
```

Comment:
Change the name and id values, as they are duplicated in the `DownloadCacheModal`. Using this changing the radio button in the download modal will only impact the code in CodeEditor.